plugins/plugin-testing-java/plugin-testing-testng/che-plugin-testing-testng-server/src/main/java/org/eclipse/che/plugin/testing/testng/server/TestNGRunner.java

```diff
@@ -47,13 +49,59 @@
  *             (Note: If the <em>runClass</em> parameter is true then <em>testngXML</em> parameter gets ignored.)
  *
  * </pre>
+ * 
  * @author Mirage Abeysekara
  */
 public class TestNGRunner implements TestRunner {
 
-    String projectPath;
-    ClassLoader projectClassLoader;
+    private String absoluteProjectPath;
```

Comment:
can we avoid use project path as field? Maybe better put it as parameters for methods